tools/SchemaManager/Commands/ApplyCommand.cs

```diff
@@ -56,7 +60,9 @@ private async Task HandlerAsync(string connectionString, Uri server, MutuallyExc
             return;
         }
 
-        await _schemaManager.ApplySchema(connectionString, server, exclusiveType, cancellationToken).ConfigureAwait(false);
+        _schemaClient.SetUri(server);
+
+        await _schemaManager.ApplySchema(connectionString, exclusiveType, cancellationToken).ConfigureAwait(false);
```

Comment:
nit: remove `await` and `async` from this method. We can simply return the `Task` itself

Reply:
Should I make this change in AvailableCommand.cs and CurrentCommand.cs as well?